src/modules/common/renderHtml.js

```diff
@@ -54,26 +55,67 @@ const renderJS = (files: FileCacheStateType) => `
 `
 
 export default (html: string, files: FileCacheStateType, theme: ThemeType) => {
+  const getSource = (fontName: string) => Platform.select({
+    ios: `local(${fontName}) format('truetype')`,
+    android: `url('file:///android_asset/fonts/${fontName}.ttf') format('truetype')`
+  })
   // language=HTML
   return `
 <html>
 <head>
   <!-- disables zooming https://stackoverflow.com/questions/44625680/disable-zoom-on-web-view-react-native-->
   <meta name="viewport" content="initial-scale=1.0, maximum-scale=1.0">
   <style>
+    @font-face {
+      font-family: 'OpenSans';
+      font-style: normal;
+      font-weight: 400;
+      src: ${getSource('OpenSans-Regular')};
+    }
+    @font-face {
+      font-family: 'OpenSans';
+      font-style: normal;
+      font-weight: 700;
+      src: ${getSource('OpenSans-Bold')};
+    }
+    @font-face {
+      font-family: 'Raleway';
+      font-style: normal;
+      font-weight: 400;
+      src: ${getSource('Raleway-Regular')};
+    }
+    @font-face {
+      font-family: 'Raleway';
+      font-style: normal;
+      font-weight: 700;
+      src: ${getSource('Raleway-Bold')};
+    }
+    @font-face {
+      font-family: 'Lateef';
+      font-style: normal;
+      font-weight: 400;
+      src: ${getSource('Lateef')};
+    }
+   
     html, body {
         margin: 0;
         padding: 0;
         
-        /*font-family: \${theme.fonts.contentFontFamily};*/   
+        font-family: ${theme.fonts.webviewFontFamilies};
         font-size: ${theme.fonts.contentFontSize};
         line-height: ${theme.fonts.contentLineHeight};
+        font-size-adjust: ${theme.fonts.fontSizeAdjust};
+        background-color: ${theme.colors.backgroundColor};
         /*\${props => props.centered && css\`
```

Comment:
Can you remember why this is here? If not remove it.

Reply:
This is for centering the (former) RemoteContent. I don't remove it now, since we might get back to this.